python/tests/pyValidator.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -x
```

Comment:
Is there something in the debug info that the user should see when running this script? If not, I would recommend removing this flag.